cmd/kola/options.go

```diff
@@ -50,6 +52,7 @@ func init() {
 	sv(&outputDir, "output-dir", "", "Temporary output directory for test data and logs")
 	sv(&kola.TorcxManifestFile, "torcx-manifest", "", "Path to a torcx manifest that should be made available to tests")
 	root.PersistentFlags().StringVarP(&kolaPlatform, "platform", "p", "qemu", "VM platform: "+strings.Join(kolaPlatforms, ", "))
+	root.PersistentFlags().StringVarP(&kolaDistro, "distro", "l", "cl", "Distribution: "+strings.Join(kolaDistros, ", "))
```

Comment:
Looking for suggestions for a better short-name

Reply:
`-b` for `distroBution`?